src/escalus_pubsub_stanza.erl

```diff
@@ -320,6 +327,15 @@ subscription_options(NodeName, User) ->
            attrs = [{<<"node">>, NodeName},
                     {<<"jid">>, escalus_utils:get_jid(User)}]}.
 
+set_subscription_options_form(NodeName, User, Children) ->
+    #xmlel{name = <<"options">>,
```

Comment:
Looks like (at least on GitHub) that the indentation is totally broken for this record. We usually format such records like follows:
```erlang
    #xmlel{name = <<"options">>,
           attrs = [{<<"node">>, NodeName},
                    {<<"jid">>, escalus_utils:get_jid(User)}],
           children = [#xmlel{name = <<"x">>,
                              attrs = [{<<"xmlns">>, <<"jabber:x:data">>},
                                       {<<"type">>, <<"submit">>}],
                              children = Children}]}.

```

Reply:
I'll fix the formatting :)